ads/_config.js

```diff
@@ -747,3 +747,14 @@ export const adConfig = {
   },
 
 };
+
+
+/**
+ * Specifies which networks can share master frames.
+ * @type {!Object<string, !Array<string>>}
+ */
+export const masterFrameAccessibleTypes = {
```

Comment:
If we do that, then we'd have to import the entire adConfig object in ampcontext-integration.js which might cause some bloat in the binary size. Should I move anyway?